css-layout-classes/styles.css

```diff
@@ -0,0 +1,30 @@
+#container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.col-2 {
+  flex-basis: calc(100% /3);
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-two-third {
+  flex-basis: calc(200% / 3);
+}
+
+.column-one-fourth {
```

Comment:
make sure you are setting the proper flex basis for this element. And this class should not have display flex on it